src/foam/nanos/auth/Address.js

```diff
@@ -219,6 +220,16 @@ foam.CLASS({
       },
       gridColumns: 6,
       validationPredicates: [
+        {
+          args: ['postalCode'],
+          predicateFactory: function(e) {
+            return e.GT(
+              foam.mlang.StringLength.create({
+                arg1: foam.nanos.auth.Address.POSTAL_CODE
+              }), 0);
```

Comment:
why the new validation predicate? Just include this GT expression within the validationPredicate below. I'm sure including 'Valid' will still get the message across to the user. The errorMessage in the vp below also includes both cases where zip code and postal code are mentioned.

Reply:
Ya was thinking this but confirmed on the ticket - it was what was desired. 'they' want different msg for this.